index.md

```diff
@@ -16,12 +18,14 @@
 - Document view of your outlines, for reading document-style Org files
   with long body text
 - Powerful support for linking between Org files
-- Org-mode integration
+- OrgMode integration with [org-mobile.el](http://orgmode.org/manual/MobileOrg.html)
```

Comment:
I checked Org's website and found the following names used synonymously:
- Org
- Org mode

in older documents the name is 'Org-mode'. I found no occurrence of 'OrgMode', though.